endersgame/widgets/streams.py

```diff
@@ -1,19 +1,10 @@
-from datetime import datetime
-
 from pydantic import BaseModel
 
-
 class Prediction(BaseModel):
     value: float
-    # TODO: Oneof?
-    t: datetime = None # absolute time
     n: int = None # relative horizon
```

Comment:
todo: `n` has been renamed to `horizon`

Reply:
`n` is not the horizon, it's the "index tick" because we are on a discrete schedule. Basically time in discrete.

Reply:
Do we want some other convention? `idx` maybe?

Reply:
Do not use abbreviations, use `index` directly.

Reply:
Peter uses `ndx` which is ok, this is what I changed to.